### DISCUSSION TITLE
Possible issue with installer on WSL & docker-desktop kubernetes?

### DISCUSSION THREAD

**Discussion by Nopzen** [gimlet-io/gimlet#350] [Q&A]
I wanted to give Gimlet a test on my local docker-desktop created kubernetes cluster using the guide here: https://gimlet.io/docs/installation. The installation seemed to get stuck / hanging for 5/10 minutes before I terminated the process with Ctrl+C

After the "first" termination the installer started to log message:

```bash
 🧐 Waiting for all four gitops kustomizations become ready, ctrl+c to abort

$ kubectl get kustomizations.kustomize.toolkit.fluxcd.io -A
error: the server doesn't have a resource type "kustomizations"
````

According to the installer no Flux CD seemed like a requirement, assuming it would be a part of the installer, if this is not the case I've clearly misunderstod requirements..

After second Ctrl+C installer terminated properly.

### Environment

**OS**: Windows 11, WSL 2
**Kubernetes**: stock docker-desktop kubernetes.

### Screenshot of issue:
![gimlet-stuck-install](https://user-images.githubusercontent.com/2326591/197248554-d51f0425-e7b8-4d4a-9127-42a4ad84b88c.png)

**Comment by laszlocph** [comment]
Hello. Flux is part of the installer. Did you apply the manifests? The blue stuff on this screenshot. Step-3 in the installer https://gimlet.io/docs/installation#kickstarting-the-gitops-automation

**Reply by Nopzen** [reply]
Never made it that far, I got stuckon this command:

```bash
curl -L -s https://get.gimlet.io | bash -s trial
```

**Reply by laszlocph** [reply]
hm. That command starts the installer, you go through step-1-2-3, then close the browser tab, then you see the script output you posted.
So for some reason you closed the tab sooner than should have. I wonder if there was an error that made you do it.

This docs describes step by step what should happen https://gimlet.io/docs/installation
This is a video walk through https://youtu.be/CPOAc95MhT4?t=362 if you are unsure about the docs

**Reply by Nopzen** [reply]
Thanks, I will have a look at this today, but i did not get not get a tab / browser start after the script started, I will try and watch the video, it very well could be a me issue, I will report back later once I've checked out the vid.
